dashboard/lib/tasks/asset_sync.rake

```diff
@@ -14,9 +14,26 @@ namespace :assets do
     app.assets = Sprockets::Railtie.build_environment(app)
     Sprockets::Railtie.build_manifest(app).compile('application.js')
   end
+
+  desc 'Copy digested assets to non-digested file paths'
+  task no_digests: :environment do
+    assets = Dir.glob("#{dashboard_dir}/public/assets/js/**/*")
+    regex = /(-{1}[a-z0-9]{32}*\.{1}){1}/
+    assets.each do |file|
+      next if File.directory?(file) || file !~ regex
+
+      source = file.split('/')
+      source.push(source.pop.gsub(regex, '.'))
+
+      non_digested = File.join(source)
```

Comment:
You can avoid the split/push/pop/join nonsense with a non-capturing group in your regex to ensure it's the end of the filename.
```ruby
regex = /-\w{32}(?:\.\w+$)/
assets.each do |file|
  next if File.directory(file) || file !~ regex
  non_digested = file.sub(regex, '')
  FileUtils.cp(file, non_digested)
end